subsys/bluetooth/services/ras/Kconfig.ras

```diff
@@ -4,10 +4,34 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config BT_RAS
-	bool
+menuconfig BT_RAS
+	bool "Ranging Service [EXPERIMENTAL]"
+	depends on BT_CHANNEL_SOUNDING
+	select BT_NRF_SERVICES
+	select EXPERIMENTAL
 	help
-	  Common Bluetooth GATT Ranging Service modules.
+	  Bluetooth GATT Ranging Service modules - RREQ and RRSP.
+
+if BT_RAS
 
 rsource "rreq/Kconfig.ras_rreq"
 rsource "rrsp/Kconfig.ras_rrsp"
+
+config BT_RAS_MAX_ANTENNA_PATHS
+	int "Maximum number of antenna paths supported"
+	default 4
+	range 1 4
+	help
+	  The number of antenna paths per step that can be stored inside RAS.
+	  Must match the capabilities of the local and peer devices.
```

Comment:
it's a nitpick but I think this only needs to be larger than `max_antenna_paths_supported` reported in LE CS Read Local Supported Capabilities, because the peer device can't use more antenna paths than what the local device supports